src/main/java/com/archisacademy/jobportal/controller/CommentController.java

```diff
@@ -0,0 +1,48 @@
+package com.archisacademy.jobportal.controller;
+
+import com.archisacademy.jobportal.dto.CommentDto;
+import com.archisacademy.jobportal.services.CommentService;
+import org.springframework.data.domain.Page;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/comments")
+public class CommentController {
+
+    private final CommentService commentService;
+
+    public CommentController(CommentService commentService) {
+        this.commentService = commentService;
+    }
+
+    @PostMapping("/create")
+    public ResponseEntity<String> createComment(@RequestBody CommentDto commentDto) {
+        return new ResponseEntity<>(commentService.createComment(commentDto), HttpStatus.CREATED);
+    }
+
+    @DeleteMapping("/delete/{id}")
+    public ResponseEntity<String> deleteComment(@PathVariable Long id,
+                                                @RequestParam String userUuid) {
+         return new ResponseEntity<>(commentService.deleteComment(id, userUuid), HttpStatus.OK);
+    }
+
+    @PutMapping("/update/{id}")
+    public ResponseEntity<String> updateComment(@PathVariable Long id,
+                                                @RequestBody CommentDto commentDto,
```

Comment:
please add userUUId inside commentDto you don't need to have a sperate request param for a single value which is wrong